src/core/unit_tests/reaction_ensemble_utils_test.cpp

```diff
@@ -39,8 +39,8 @@ BOOST_AUTO_TEST_CASE(average_list_of_allowed_entries_test) {
   using namespace ReactionEnsemble;
   constexpr double tol = 100 * std::numeric_limits<double>::epsilon();
 
-  BOOST_CHECK_CLOSE(
```

Comment:
Why is RE stuff in this diff?

Reply:
there is an issue in the RE unit test that was not caught by CI on the dev branch (it was detected by the manual `intel:19` job)